browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/suggestions/editurl/BraveEditUrlSuggestionProcessor.java

```diff
@@ -39,8 +39,8 @@ public boolean doesProcessSuggestion(@NonNull AutocompleteMatch suggestion, int
         Tab activeTab = mTabSupplier.get();
         if (position == 0
                 && activeTab != null
-                && (suggestion.getType() == OmniboxSuggestionType.URL_WHAT_YOU_TYPED
-                                && tabMatchesSuggestion(activeTab, suggestion)
+                && ((suggestion.getType() == OmniboxSuggestionType.URL_WHAT_YOU_TYPED
+                                && tabMatchesSuggestion(activeTab, suggestion))
```

Comment:
This is changing the behavior - just want to make sure that is expected. ex: before `activeTab` would need to pass `!= null` before it would call below (`activeTab.getUrl`). Now, it would jump to that OR condition and it could be null

Reply:
This change only explicetly groups this part

Reply:
I'll split thif `if` statement in 2 `if`s to make the logic more explicite.

Reply:
Fixed here https://github.com/brave/brave-core/commit/4b340b8f22dc9fe197218dbf075888a0e4d205fd

Reply:
Perfect! 😄👍 Thanks @samartnik